### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by lsetiawan**
## Overview

This PR updates the `setup.py` in preparation of new release. This also setup `versioneer`. It is a part of #59.

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+flake8
```
Comment (ocefpaf) [inline requirements-dev.txt]: :+1:

**Comment by lsetiawan** [comment]
@ocefpaf If you have sometime it would be great if you could review this PR to make sure, I'm doing things right with `versioneer` and `travis`. Thank you 😄

File: odm2rest/__init__.py [hunk 2]
```diff
@@ -1 +1,5 @@
-API_VERSION = 'v1'
\ No newline at end of file
+API_VERSION = 'v1'
+
+from ._version import get_versions
+__version__ = get_versions()['version']
```
Comment (ocefpaf) [inline odm2rest/__init__.py]: :+1:

**Review by ocefpaf — APPROVED** [review]

**Comment by ocefpaf** [comment]
> @ocefpaf If you have sometime it would be great if you could review this PR to make sure, I'm doing things right with `versioneer` and `travis`. Thank you 😄 

Everything looks good to me!

**Comment by lsetiawan** [comment]
Thank you for reviewing <3

Merging now.

**Pull request merged**